massdriver-application-azure-function-app/subnet.tf

```diff
@@ -0,0 +1,22 @@
+locals {
+  split_id            = split("/", var.virtual_network_id)
+  network_name        = element(local.split_id, index(local.split_id, "virtualNetworks") + 1)
+  resource_group_name = element(local.split_id, index(local.split_id, "resourceGroups") + 1)
+}
```

Comment:
```suggestion
  network_name        = regex(".*/virtual[n|N]etworks/(.*)$", var.virtual_network_id)[0]
  resource_group_name = regex(".*/resource[g|G]roups/(.*)/providers", var.virtual_network_id)[0]
}
```

Reply:
Tested this locally, it works. Will commit with some other changes here soon.